packages/gatsby-theme-store/src/sdk/orderForm/useToast.ts

```diff
@@ -3,17 +3,17 @@ import { useCallback } from 'react'
 import { useToast as useToastFromProvider } from '../toast/useToast'
 
 export const useToast = () => {
-  const { showToast, isVisible } = useToastFromProvider()
+  const { showToast } = useToastFromProvider()
 
   const showToastWrapped = useCallback(
     (message: string) => {
-      showToast({ message, duration: 6000, type: 'error' })
+      showToast({ content: message, duration: 5000, type: 'error' })
```

Comment:
Was the message being ignored before, or does this function accept an object with either `message` or `content`?